WordPress/gutenberg
# [Mobile] Cover: Add Media Button when only colour is selected

**Fixes** https://github.com/WordPress/gutenberg/issues/23877 (**parent** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2275)

## Related PRs:
This PR depends on https://github.com/WordPress/gutenberg/pull/23870

* `gutenberg-mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/2513
* **Depends on** Create cover block with custom colour https://github.com/WordPress/gutenberg/pull/23870
* Create cover block with only colour from a default palette https://github.com/WordPress/gutenberg/pull/23833
* Clear media button in block settings https://github.com/WordPress/gutenberg/pull/23794

## Description
An image icon button has been added at the top-left of the component. When pressed the add media options are presented.

## How has this been tested?
* Add a cover block
* Press on a colour from the palette
* Verify that a new cover block with only colour is created
* Verify that the new icon is presented at the top-left of the component
* Tap on the icon and verify that the add media options are presented
* Add a media and verify that it is applied in the background

## Screenshots

|Screenshot|Functionality|
|---|---|
|![Simulator Screen Shot - iPhone 11 - 2020-07-10 at 18 51 16](https://user-images.githubusercontent.com/304044/87174203-826e8c80-c2df-11ea-8790-c8a623805be0.png)|![app](https://user-images.githubusercontent.com/304044/87912606-589f2d80-ca76-11ea-9e05-6bf0388b3f6e.gif)|

## Types of changes
Enhancement

## Checklist:
